app/views/createAccount/confirmation.scala.html

```diff
@@ -0,0 +1,58 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
```

Comment:
It's 2022 🙂

Reply:
I have updated the year

Reply:
I was joking, but thanks. All our twirl template headers are 2021 across all services. Something we will probably fix somehow, sometime.